css/fullcalendar.scss

```diff
@@ -171,3 +171,24 @@
 .fc * {
 	box-sizing: content-box !important;
 }
+
+// Reminder icon on events with alarms set
+.fc-event-nc-alarms {
+	.icon-event-reminder {
+		background-color: inherit;
+		background-position: right;
+		position: absolute;
+		top: 0;
+		right: 0;
+		&--light {
+			background-image: url('/svg/calendar/reminder?color=ffffff')
```

Comment:
This will break if there is index.php in the url or if Nextcloud is installed in a subdirectory.

Reply:
```suggestion
			@include icon-color('reminder', 'calendar', '#fffffe', 1);
```